app/screens/entry/entry.js

```diff
@@ -183,16 +185,16 @@ export default class Entry extends PureComponent {
             const notificationData = notification || app.replyNotificationData;
             const {data, text, badge, completed} = notificationData;
 
-            onPushNotificationReply(data, text, badge, completed);
+            if (completed) {
+                onPushNotificationReply(data, text, badge, completed);
+            } else {
+                await store.dispatch(loadFromPushNotification(notification));
```

Comment:
I'm not following what's going on here.  Maybe a comment for documentation would help others in the future.

From what I can tell this is coming from the headless JS code.
But completed is a function so should we be calling `completed()` above?

Or change 
`completed: () => true` to `completed: true`